controller/src/pid.cpp

```diff
@@ -42,6 +43,10 @@ enum class pid_fsm_state {
 
 void pid_init() {
 
+  //@TODO: Raw access that needs to be updated to reference the new modules
```

Comment:
Could we state explicitly what we mean by "the new modules"?  Are we referring to `sensors`?